CMakeLists.txt

```diff
@@ -179,25 +179,23 @@ target_link_libraries(${PROJECT_NAME}_node
 # )
 
 ## Mark executables and/or libraries for installation
-# install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_node
-#   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-#   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-#   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-# )
+install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_node
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
 
 ## Mark cpp header files for installation
-# install(DIRECTORY include/${PROJECT_NAME}/
-#   DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
-#   FILES_MATCHING PATTERN "*.h"
-#   PATTERN ".svn" EXCLUDE
-# )
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+  FILES_MATCHING PATTERN "*.*"
```

Comment:
Could you change this and the next line back to:
```
#   FILES_MATCHING PATTERN "*.h"
#   PATTERN ".svn" EXCLUDE
```

Reply:
Correct me if I am wrong:
(1) In my workspace, including the line
```
FILES_MATCHING PATTERN "*.*"
```
installs all *.hpp and *.data. However, excluding the line also installs all *.hpp and *.data, so it seems to be not required here. This would also be in line with the [catkin documentation](http://docs.ros.org/kinetic/api/catkin/html/howto/format2/building_libraries.html) and these [OSRF slides](https://www.osrfoundation.org/wordpress2/wp-content/uploads/2015/04/ROSCON-2013-Understanding-and-using-catkin-and-bloom.pdf), where it says that this line can be used to skip hidden files or folders.

(2) Since it is not a Subversion repository, the line
```
PATTERN ".svn" EXCLUDE
```
is not required.

If you agree, please change these two lines back to the original ones
```
#   FILES_MATCHING PATTERN "*.h"
#   PATTERN ".svn" EXCLUDE
```